.env.gold

```
@@ -0,0 +1,5 @@
+NEXT_PUBLIC_PROTOCOL_NAME=GLUON GOLD
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)